src/pages/Proposal.tsx

```diff
@@ -643,11 +653,8 @@ const ProposalPage = observer(() => {
               })}
           </span>
           <span style={{ width: '50%', textAlign: 'center', color: 'red' }}>
-            {proposal.negativeVotes
-              .times('100')
-              .div(totalRepAtProposalCreation)
-              .toFixed(2)}{' '}
-            %<AmountBadge color="red">{negativeVotesCount}</AmountBadge>
+            {`${positiveVotes}%`}

```

Comment:
this should be negativeVotes, found this bug when I went to vote in xdai and I saw that the  negative voting amout suddenly changed after I click on voting